### ISSUE TITLE
Permanent (very long) shakes disease from trait combo

### ISSUE THREAD

**Issue by Pooshmeu** [CleverRaven/Cataclysm-DDA#48520]
Describe the bug

"shakes" disease for ever.

Steps To Reproduce

1-create character with traits, "addictive personality", "lightweight", and start with "heroin addict".
2- first 2-3 days wherever you are on withdrawl, use the heroin.
3- overcome the addiction, and now character still has the "shakes" disease.

Expected behavior

long after overcoming the addiction, I still got the shakes disease, my character shouldn't have -4dex and -1str a month after overcoming heroin addiction.

Versions and configuration

I don't have the character anymore, but I remember generating the "effect_list.output" and I had the shakes disease stacked to last 9 weeks and 1 day, my character didn't have any other disease (like tetanus for example), and my character had already stopped having "opiate withdrawl" disease for about 2-3 ingame weeks.

 - OS: w10
 - Game Version: experimental version from about 2 weeks ago.
 - Graphics version: Tiles, UDP
 - Ingame language: English
 - Mods loaded: A lot of mods, but supposedly none messed with drugs or diseases.

Additional context

**Comment by OliveChukar** [comment]
Tested on 0.E-10395-g4525b4f [64-bit] with Dark Days Ahead [dda] only no mods. Started game as a heroin addict with addictive personality lightweight and everything else default. Used heroin every time withdrawal started and ran out on the first day. All addiction problems other then shakes ended after 12 days. I waited two more days then gave up and checked the character data for shakes duration it was 7038386. If I am understanding the numbers correctly this would be 81 days.
